code/workspaces/test/test-results/10893_STS.md

```diff
@@ -459,13 +470,67 @@ System testing is carried out as part of every sprint to ensure the completed st
->>> | 3 | View the response for the second to last api call | Contains the projects the user has access to and the admin status:  "system:catalogue:admin" | PASS |
+>>> | 3 | View the response for the second to last api call | Contains the projects the user has access to and the admin status:  "system:catalogue:admin". It also contains the catalogue status e.g. catalogue publisher | PASS |
 >>> | 4 | Log out of Datalabs | User is logged out successfully | PASS |
 >>> | 5 | Log in as a user that is not an admin | User is logged in and is taken to the /projects page | PASS |
 >>> | 6 | Repeat steps 1-3 as the non-admin user | Api response only contains the projects the user has access to. It does not contain admin information | PASS |
 ---
 
+>>>###### **TC2.7.4 Admin user has access to user management page**
+>>> | **JIRA ID(s):** | [NERCDL-664](https://jira.ceh.ac.uk/browse/NERCDL-664?jql=key%20in%20(NERCDL-664)) |OVERALL RESULT: | PASS |
+>>> | ------------ | --------- | --------- | ------|
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 | Log into Datalabs as an admin user | User successfully logs in and there is an admin button in the top navigation menu | PASS |
+>>> | 2 | Click on the admin button | User is taken to the resources admin page. There is a "Users" tab on the left side of the page | PASS |
+>>> | 3 | Click on the "Users" tab | User is taken to the user management page | PASS |
+---
+
+>>>###### **TC2.7.4 User management page filters functionality tests**
+>>> | **JIRA ID(s):** | [NERCDL-664](https://jira.ceh.ac.uk/browse/NERCDL-664?jql=key%20in%20(NERCDL-664)) |OVERALL RESULT: | PASS |
+>>> | ------------ | --------- | --------- | ------|
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 | Whilst on the user management page, click on the users field. **Note:** Ensure that the logged in account is a part of multiple projects and is the project admin for at least one project | A drop down of users are displayed. The current logged in user should be displayed here | PASS |
+>>> | 2 | Start to type in a known username | The drop-down list is filtered | PASS |
+>>> | 3 | Select a user by clicking on an item from the drop-down list | User is now displayed in the user field and only items relating to the user are shown on the page | PASS |
+>>> | 4 | Enter a username that is not a user in the Datalabs system | "No options" message is shown in the drop-down | PASS |
+>>> | 5 | Click on the X next to the selected user | User is removed from the field and the page is no longer filtered against the one user | PASS |
+>>> | 6 | View the filter checkboxes | There are various admin, user and owner filters | PASS |
+>>> | 7 | Select some filters e.g. instance admin and project admin | The page is filtered to only show users that meet these filter requirements. Checkboxes are clearly marked as selected | PASS |

```

Comment:
Should be "The page is filtered to only show users that meet one of these filter requirements." i.e. it will show users who are either instance admins or project admins; you don't have to be an instance admin and a project admin to appear.

Reply:
Sorry, probably even more clearer to say "that meet at least one of these filter requirements".